### PR TITLE
[ui] Show unaffiliated teams

### PR DESCRIPTION
**Pull request by evamillan**
This PR adds a table for groups (teams that aren't affiliated to any organization) on the main view. It uses the `OrganizationTable` component since the functionality and look is almost identical, but adds the `isGroup` prop to manage the differences like hiding the domains or passing different parameters to queries and mutations.

### CODE REVIEW

File: ui/src/components/TeamModal.vue [hunk 1]
```diff
@@ -189,7 +197,8 @@ export default {
     async delete(item) {
       this.closeDialog();
       try {
-        const response = await this.deleteTeam(item.name, this.organization);
+        const organization = this.isUnaffiliated ? null : this.parent;
```
Comment (sduenas) [inline ui/src/components/TeamModal.vue:200]: Shouldn't it be better to have `this.isAffiliated`? The logic might be clear but I'm not sure if we use `isUnaffiliated` somewhere else?

**Review by sduenas — COMMENTED** [review]
LGTM, however check my comment about `unaffiliated`var.

File: ui/src/components/TeamModal.vue (hunk 1 above)
Comment (evamillan) [reply ui/src/components/TeamModal.vue]: I've changed it to `isGroup`, which is the variable used in the other components.

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**